CHANGELOG.rst

```diff
@@ -4,6 +4,138 @@ SeqAn Changelog
 This file summarizes the changes to the SeqAn library and apps.
 
 
+Release 2.1.0
+~~~~~~~~~~~~~
+
+Major release with many new features and applications. 
+Note, the majority of the modules are backward compatible to the previous version.
+For a complete list of changes visit `GitHub <https://github.com/seqan/seqan/pulls?q=is%3Apr+is%3Amerged++milestone%3A%22Release+2.1.0%22+>`_.
+
+Library Updates and Selected Bugfixes
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+- Apps:
+    - Yara: fixed warnings, build errors and bugs, updated test files
+    - Yara: new features (compute mapping qualities, estimate distribution of paired-end insert sizes)
+    - Yara: follow SAM recommended practices for paired-end reads
+    - T-Coffee: new feature ``deep coffee`` (alignining several hundred sequences)
+    - Lambda: simplified find interface
+    - Gustaf: introduced two phase breakpoint combination; updated readme and help messages
+    - Removed old apps: Razers1, Razers2, Flexbar and SeqCons in favor of newer releases
+
+- Alignments:
+    - added feature to count gaps to the left a of a position/iterator
+    - disallow wrong use of scoring scheme for Hirschberg algorithm
+    - extended AlignmentStats by number of gaps and length of the alignment
+    - fixed evaluation of alignment
+    - using gaps for integrateAlign and align_extend
+
+- BLAST:
+    - BLAST module with e-values, supporting BLAST tabular (with and without comments) and BLAST reports (pairwise)
```

Comment:
replace with:

BLAST (new module):
  - E-Value statistics, including precomputed constants, bit-score and e-value calculation for alignments
  - Support for reading and writing BLAST Tabular files (with and without comments)
  - Support for writing BLAST Report files